### PR TITLE
reduce web ci test time

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### What kind of change does this PR introduce? 

- [ ] Bug
- [X] Feature
- [ ] Docs
- [ ] Other(s)

### Why was this change needed?

At the moment the ci test run takes a lot of time in this PR i will try to reduce it.

### CODE REVIEW

**Comment by djabarovgeorge** [comment]
It's a WIP so you can ignore this PR at the moment, it is not a draft PR because I want to trigger the ci pipeline.

File: apps/web/src/App.tsx [hunk 1]
```diff
@@ -48,10 +48,11 @@ if (SENTRY_DSN) {
   });
 }
 
-const defaultQueryFn = async ({ queryKey }: { queryKey: string }) => {
-  const response = await api.get(`${queryKey[0]}`);
+// eslint-disable-next-line no-console
+console.log('trigger web ci test');
```
Comment (scopsy) [inline apps/web/src/App.tsx:52]: I think that left here by mistake :)

File: .github/workflows/reusable-web-e2e.yml [hunk 2]
```diff
@@ -123,26 +123,6 @@ jobs:
...
-          component: true
-          record: true
-          install: false
-          config-file: cypress.config.ts
-
-      - uses: actions/upload-artifact@v1
-        if: failure()
-        with:
-          name: cypress-screenshots
-          path: apps/web/cypress/screenshots
```
Comment (scopsy) [inline .github/workflows/reusable-web-e2e.yml:144]: Should we return this to here? so the web e2e screenshots will be uploaded too

**Review by scopsy — APPROVED** [review]

File: apps/web/src/App.tsx (hunk 1 above)
Comment (djabarovgeorge) [reply apps/web/src/App.tsx:52]: hahah yes did it in the first place in order to trigger the ci test, I will remove it.
I probably will create a new PR with rebased commit, this PR is too messy in my opinion.

File: .github/workflows/reusable-web-e2e.yml (hunk 2 above)
Comment (djabarovgeorge) [reply .github/workflows/reusable-web-e2e.yml:144]: oh, you are totally right I removed it by mistake while removing the component test.

Comment (djabarovgeorge) [reply .github/workflows/reusable-web-e2e.yml:144]: good catch!

**Comment by djabarovgeorge** [comment]
New clean branch/pr https://github.com/novuhq/novu/pull/1978

**Pull request closed without merging**